src/stores/ConfigController.ts

```diff
@@ -1,6 +1,25 @@
+import { API_HOST } from '#src/config';
+import { IS_PROD_MODE } from '#src/utils/common';
 import { useConfigStore } from '#src/stores/ConfigStore';
 import { getAdSchedule } from '#src/services/api.service';
 
+const envQueryKey = 'env';
+
+export const initializeHost = (searchParams: URLSearchParams) => {
+  const query = searchParams.get(envQueryKey);
```

Comment:
I think it would make sense to move the searchParam logic into `getConfigSource` in configOverride.ts (which would return `{configSource: string, host: string}`